ETH-Merge-Bots/Milestones-in-TTD/src/eth.config.ts

```diff
@@ -0,0 +1,19 @@
+import BigNumber from "bignumber.js";
+
+export const TERMINAL_TOTAL_DIFFICULTY: BigNumber = new BigNumber("58750000000000000000000");
+
+// Days left for the merge
+export const MILESTONES = {
+  LOW: 20,
+  MEDIUM: 15,
+  HIGH: 10,
+  CRITICAL: 5,
+};
+
+export const ETH_BLOCK_DATA = {
+  avgBlockTime: 13.46, // average block time between Aug 13th - Aug 19th
+  blockNumberAWeek: 44236, // total number of blocks mined in between Aug 13th - Aug 19th
```

Comment:
we could probably use etherscan api to fetch this data. but since I've never used it before, I didn't want to lose time. but I'll learn about it 👍 and yes, I can include the most recent info tomorrow. keeping this open 👍

Reply:
done 👍